clang/lib/3C/RewriteUtils.cpp

```diff
@@ -418,7 +418,7 @@ void RewriteConsumer::HandleTranslationUnit(ASTContext &Context) {
   CheckedRegionFinder CRF(&Context, R, Info, Seen, NodeMap, WarnRootCause);
   CheckedRegionAdder CRA(&Context, R, NodeMap);
   CastLocatorVisitor CLV(&Context);
-  CastPlacementVisitor ECPV(&Context, Info, R, CLV);
+  CastPlacementVisitor ECPV(&Context, Info, R, CLV.getExprsWithCast());
```

Comment:
So you are passing in the set, and then later it gets populated, when you run `CLV`? Seems a little odd; you might mention this is happening.